### DISCUSSION TITLE
Aiuto su indirizzamento voce echo da script

### DISCUSSION THREAD

**Discussion by Paoloc66** [indomus/forum#1884] [HUB: Home Assistant]
Probabilmente il titolo non va bene, ma non sapevo cosa mettere.
Mi chiedevo se era fattibile ed eventualmente come fare a reindirizzare l'output vocale sullo stesso dispositivo in cui viene fatta la richiesta. Mi spiego meglio...
Situazione: 
- Ho l'integrazione con alexa
- ho un dispositivo Echo in ogni camera della casa
- Ho creato un semplice script su HA che mi legge la temperatura della casa presa dai sensori Aqara su **ECHO DELLA SALA**
- Su Alexa ho creato una routine che quando dico "che temperatura c'è in casa" mi lancia questo script 
Funziona tutto perfettamente.

Vorrei fare in modo che se chiedo la temperatura ad alexa dall'echo della camera mi faccia partire la routine ma che mi risponda sempre sull'echo della camera e non sull'echo in sala.
Consigli?

**Comment by Larry-78** [comment]
Ciao, dovresti utilizzare il servizio `notify.alexa_media_last_called`. Puoi trovare tutte le informazioni nella documentazione ufficiale [qui](https://github.com/custom-components/alexa_media_player/wiki#last-called-device)

**Reply by Paoloc66** [reply]
Ottimo Grazie!!
